src/pages/Graph/GraphToolbar/GraphSettings.tsx

```diff
@@ -90,7 +138,34 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps, GraphSetting
       toggleUnusedNodes
     } = this.props;
 
-    const visibilityLayers: VisibilityLayersType[] = [
+    const edgeLabelOptions: DisplayOptionType[] = [
+      {
+        id: EdgeLabelMode.NONE,
+        labelText: _.capitalize(_.startCase(EdgeLabelMode.NONE)),
```

Comment:
Nitpick, is the `_capitalize` over all here really needed?
`_.startCase('requestsPerSecond');` yields: `Requests Per Second`

Reply:
smarty.  changed.